src/app/core/services/session-fix-confirmation/session-fix-confirmation.component.ts

```diff
@@ -18,7 +18,7 @@ export class SessionFixConfirmationComponent implements OnInit {
     @Inject(MAT_DIALOG_DATA) public data: RepositoryData) {}
 
   onNoClick(): void {
```

Comment:
We can actually remove this `onNoClick` method, since the "No" button already has a `mat-dialog-close` directive, with a `false` result.

Reply:
I see, I didn't realise that😅  I have removed the function!